mycroft/client/wifisetup/main.py

```diff
@@ -61,17 +88,62 @@ def sysctrl(*args):
     return cli('systemctl', *args)
 
 
-class WebServer(Thread):
-    DIR = dirname(realpath(__file__))
+class MycroftHTTPRequestHandler(SimpleHTTPRequestHandler):
```

Comment:
I had it commented out until I was able to test it.  With an iOS device, this makes the portal appear automatically when the device connects to the MYCROFT network.